yolort/utils/update_module_state.py

```diff
@@ -12,6 +12,8 @@ def update_module_state_from_ultralytics(
     arch: str = 'yolov5s',
     version: str = 'v4.0',
     num_classes: int = 80,
+    pretrained_file: str = None,
```

Comment:
I think this parameter name should contain 'custom' information, so that users can easily know how to use this parameter. We can follow ultralytics's naming rules [here](https://github.com/ultralytics/yolov5/blob/ed2c74218d6d46605cc5fa68ce9bd6ece213abe4/hubconf.py#L112), and insert 'custom' to it, maybe `custom_path_or_model` is one option?